applications/GeoMechanicsApplication/custom_workflows/dgeoflow.cpp

```diff
@@ -216,33 +214,31 @@ namespace Kratos
         std::size_t found = name.find_last_of('.');
         std::string subname = name.substr(found + 1);
         ModelPart &part = model_part.GetSubModelPart(subname);
-        processes.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_X,
+        mProcesses.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_X,
                                                                                                          0.0, 0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED)));
```

Comment:
Since `make_shared` receives a list of arguments that it will pass on to the constructor of class `ApplyConstantScalarValueProcess`, there is no need to create a temporary object of this type. In other words, you could simplify this as follows:
```suggestion
        mProcesses.push_back(make_shared<ApplyConstantScalarValueProcess>(part, VOLUME_ACCELERATION_X,
                                                                                                         0.0, 0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED));
```